test/unit/full_text_search/issue_test.rb

```diff
@@ -44,6 +44,37 @@ def test_save
                    targets.collect {|target| target.attributes.except("id")})
     end
 
+    test "Test that when you update the tag_ids of an issue, journal tag_ids is also updated" do
```

Comment:
Can we simplify this?

```suggestion
    def test_save_journal_status
```